optional_plugins/html/avocado_result_html/__init__.py

```diff
@@ -267,6 +267,8 @@ def _render(self, result, output_path):
             report_file.write(report_contents)
 
     def render(self, result, job):
+        if job.status == "RUNNING":
```

Comment:
It is because I removed the commit which disables the `render` when the job is RUNNING. I prefer treating this inside the results as it allows producing output even on such failures (eg. UI which says what happened, or even html plugin which says there were no test references specified...)